pkgs/by-name/yt/ytermusic/package.nix

```diff
@@ -15,18 +15,15 @@ rustPlatform.buildRustPackage rec {
   src = fetchFromGitHub {
     owner = "ccgauche";
     repo = "ytermusic";
-    rev = "beta-${version}";
+    tag = "beta-${version}";
     hash = "sha256-nu/vedQNs5TgCG1v5qwwDTnFTyXCS2KnLVrnEhCtzCs=";
   };
 
-  cargoLock = {
-    lockFile = ./Cargo.lock;
-    outputHashes = {
-      "rusty_ytdl-0.6.6" = "sha256-htXD8v9Yd7S0iLjP6iZu94tP5KO5vbmkdUybqA7OtlU=";
-      "symphonia-0.5.4" = "sha256-uf0BbpqtlpZhsnV7Cm8egxjb/fXSINsOANTjDUQ4U9M=";
-    };
-  };
-  postPatch = "cp ${./Cargo.lock} Cargo.lock";
+  cargoPatches = [ ./time-crate.patch ];
```

Comment:
Usually, a comment is added explaining why a patch is required and if it needs to be removed at the next bump.